src/main/java/plugily/projects/buildbattle/commands/arguments/admin/ThemeArgument.java

```diff
@@ -0,0 +1,167 @@
+/*
+ *
+ * BuildBattle - Ultimate building competition minigame
+ * Copyright (C) 2021 Plugily Projects - maintained by Tigerpanzer_02, 2Wild4You and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package plugily.projects.buildbattle.commands.arguments.admin;
+
+import java.util.Locale;
+
+import org.bukkit.ChatColor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+import plugily.projects.buildbattle.Main;
+import plugily.projects.buildbattle.commands.arguments.ArgumentsRegistry;
+import plugily.projects.buildbattle.handlers.themes.ThemeManager;
+import plugily.projects.minigamesbox.classic.commands.arguments.data.CommandArgument;
+import plugily.projects.minigamesbox.classic.commands.arguments.data.LabelData;
+import plugily.projects.minigamesbox.classic.commands.arguments.data.LabeledCommandArgument;
+import plugily.projects.minigamesbox.classic.handlers.language.MessageBuilder;
+
+public final class ThemeArgument {
+
+  public ThemeArgument(ArgumentsRegistry registry) {
+    registry.mapArgument("buildbattleadmin", new LabeledCommandArgument("theme", "buildbattle.admin.theme", CommandArgument.ExecutorType.PLAYER,
+        new LabelData("/bba theme &6add/remove <theme> <gameType>", "/bba theme",
+        "&7Theme commands to add or remove themes for the specified game type\n&6Permission: &7buildbattle.admin.theme")) {
+      @Override
+      public void execute(CommandSender sender, String[] args) {
+        if(registry.getPlugin().getArenaRegistry().isInArena((Player) sender)) {
+          // TODO translatable
+          new MessageBuilder("&cYou are currently playing in an arena. You can not add a new theme while the game is running.").prefix().send(sender);
```

Comment:
like why do we need to block it while ingame? I would allow it as we also allow other commands while "theoretically ingame"